addons/survey/views/survey_survey_views.xml

```diff
@@ -149,49 +148,65 @@
                 <field name="title" />
                 <field name="answer_done_count" />
                 <field name="certificate" />
+                <field name="scoring_type" />
                 <field name="color" />
                 <field name="access_mode"/>
                 <field name="activity_ids" />
                 <field name="activity_state" />
+                <field name="success_ratio"/>
                 <templates>
                     <div t-name="kanban-box" t-attf-class="oe_kanban_color_#{kanban_getcolor(record.color.raw_value)} oe_kanban_card oe_kanban_global_click o_kanban_card_survey">
+                        <div t-if="record.certificate.raw_value" title="Certification" class="certification_icon"><div class="fa fa-lg fa-trophy"></div></div>
```

Comment:
I would prefer :
```
<div t-if="record.certificate.raw_value" title="Certification" class="certification_icon">
    <span class="fa fa-lg fa-trophy"/>
</div>
```
as all in one line is quite long. Also, we prefer to use span tag to display fa icon that doesn't need a label. (i tag otherwise)